src/Main.java

```diff
@@ -1,5 +1,79 @@
+import java.util.Arrays;
+
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
+
+        System.out.println("Задание 1");
+        int[] numbers = new int[4];
+        for (int i = 1; i < numbers.length; i++) {
+            System.out.println(i);
+        }
+        double[] fraction = {1.57, 7.654, 9.986};
+        for (int x = 0; x < 3; x++) {
```

Comment:
```suggestion
        for (int x = 0; x < fraction.length; x++) {
```
используем длину массива, а не число с потолка